app/views/oaisys/pmh/list_sets.xml.builder

```diff
@@ -0,0 +1,34 @@
+xml.push_deferred_attribute('xmlns:dc': 'http://purl.org/dc/elements/1.1/',
+                            'xmlns:oai-id': 'http://www.openarchives.org/OAI/2.0/oai-identifier',
+                            'xmlns': 'http://www.openarchives.org/OAI/2.0/',
+                            'xmlns:etd_ms': 'http://www.ndltd.org/standards/metadata/etdms/1.0/',
+                            'xmlns:atom': 'http://www.w3.org/2005/Atom',
+                            'xmlns:rdfs': 'http://www.w3.org/2000/01/rdf-schema#',
+                            'xmlns:rdf': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#',
+                            'xmlns:dcterms': 'http://purl.org/dc/terms/',
+                            'xmlns:oreatom': 'http://www.openarchives.org/ore/atom/',
+                            'xmlns:oai_dc': 'http://www.openarchives.org/OAI/2.0/oai_dc/',
+                            'xmlns:xsi': 'http://www.w3.org/2001/XMLSchema-instance',
+                            'xsi:schemaLocation': 'http://www.openarchives.org/OAI/2.0/ '\
+                            'http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd '\
+                            'http://www.openarchives.org/OAI/2.0/oai-identifier '\
+                            'http://www.openarchives.org/OAI/2.0/oai-identifier.xsd '\
+                            'http://www.openarchives.org/OAI/2.0/oai_dc/ '\
+                            'http://www.openarchives.org/OAI/2.0/oai_dc.xsd '\
+                            'http://www.ndltd.org/standards/metadata/etdms/1.0/ '\
+                            'http://www.ndltd.org/standards/metadata/etdms/1-0/etdms.xsd '\
+                            'http://www.w3.org/2005/Atom')
+
+xml.tag!('request', parameters, 'https://era.library.ualberta.ca/oai')
+xml.ListSets do
+  sets.each do |top_level_sets_id_and_id, title, description|
```

Comment:
`top_level_sets_id_and_id` is a bit of a mouthful. Not 100% sure I have a better suggestion right now, but that's partially because I'm not sure I know what I expect is contained in something named this.

Reply:
Fair, renamed it to something a lot shorter and more representative (full_set_id). It's the community_id:collection:id.